mmengine/runner/runner.py

```diff
@@ -690,6 +690,41 @@ def set_randomness(self,
             deterministic=deterministic,
             diff_rank_seed=diff_rank_seed)
 
+    def set_torch_cfg(self, torch_cfg: dict) -> None:
+        """Set torch variable to by config.
+
+        Args:
+            torch_cfg (dict): Contains key-value pair which defines the names
+                of torch variables and corresponding values.
+
+        Examples:
+            >>> torch_cfg = dict(
+            >>>    backends.cuda.matmul.allow_tf32=True,
+            >>>    torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction=True
+            >>>    )
+        """  # noqa: E501
+        for attributes, value in torch_cfg.items():
+            assert isinstance(attributes, str)
+            if attributes in [
+                    'backends.cudnn.benchmark', 'backends.cudnn.deterministic'
+            ]:
+                raise ValueError(
+                    'backends.cudnn.benchmark or backends.cudnn.deterministic '
+                    'should be set in `randomness_cfg`, rather than '
+                    '`torch_cfg`')
+            attributes_list = attributes.split('.')
+            modules = attributes_list[:-1]
+            attribute = attributes_list[-1]
+            target_module = torch
+            try:
+                for module in modules:
+                    target_module = getattr(target_module, module)
+                setattr(target_module, attribute, value)
```

Comment:
Is `setattr` suitable for all conditions?
How about using `eval(str)`?

Reply:
What's the scope of this function? If we want to support other environments not only for torch, and enable/disable some functions by calling some interfaces, we've to use `eval(str)`. But it also means we need to write the config like a code, is this what we expected?